Subject: urlfetcher: add riscv64 architecture for Debian
From: Heinrich Schuchardt heinrich.schuchardt@canonical.com Mon Mar 3 11:52:58 2025 +0100
Date: Sat Mar 8 12:59:18 2025 +0100:
Git: ccfe4a0abc0dc37f6ecd367d2c3b3bb1f55649f7

Add riscv64 to the list of Debian architectures.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>

--- a/virtinst/install/urldetect.py
+++ b/virtinst/install/urldetect.py
@@ -692,7 +692,7 @@ class _DebianDistro(_DistroTree):
 
         # Check for standard arch strings which will be
         # in the URI name for --location $ISO mounts
-        for arch in ["i386", "amd64", "x86_64", "arm64"]:
+        for arch in ["i386", "amd64", "x86_64", "arm64", "riscv64"]:
             if arch in self.uri:
                 log.debug("Found treearch=%s in uri", arch)
                 if arch == "x86_64":
